kitchen.dokken.yml

```diff
@@ -39,47 +41,52 @@ platforms:
   driver:
     image: dokken/centos-6
     pid_one_command: /sbin/init
-
 - name: centos-7
   driver:
     image: dokken/centos-7
     pid_one_command: /usr/lib/systemd/systemd
-
-- name: debian-8
+- name: centos-8
   driver:
-    image: dokken/debian-8
-    pid_one_command: /bin/systemd
-    intermediate_instructions:
-      - RUN /usr/bin/apt-get update -y
+    image: dokken/centos-8
+    pid_one_command: /usr/lib/systemd/systemd
 
 - name: debian-9
   driver:
     image: dokken/debian-9
     pid_one_command: /bin/systemd
     intermediate_instructions:
       - RUN /usr/bin/apt-get update -y
-
 - name: debian-10
   driver:
     image: dokken/debian-10
     pid_one_command: /bin/systemd
     intermediate_instructions:
       - RUN /usr/bin/apt-get update -y
 
-- name: fedora-29
+- name: fedora-30
   driver:
-    image: dokken/fedora-29
+    image: dokken/fedora-30
+    pid_one_command: /usr/lib/systemd/systemd
+- name: fedora-31
+  driver:
+    image: dokken/fedora-31
     pid_one_command: /usr/lib/systemd/systemd
 
 - name: oraclelinux-6
   driver:
     image: dokken/oraclelinux-6
     pid_one_command: /sbin/init
-
 - name: oraclelinux-7
   driver:
     image: dokken/oraclelinux-7
     pid_one_command: /usr/lib/systemd/systemd
+# [2020-08-12T17:38:38+00:00] FATAL: RuntimeError: dnf_package[openssh-server] (ssh-hardening::server line 47) had an error: RuntimeError: dnf-helper.py had stderr output:
```

Comment:
Is this just leaving a note for why Oracle 8 isn't enabled? Suggest adding a TODO line to make that clear